libexec/tfenv-list-remote

```diff
@@ -73,8 +73,20 @@ remote_versions="$(curlw -sSf "${TFENV_REMOTE}/terraform/")" \
 
 #log 'debug' "Remote versions available: ${remote_versions}"; # Even in debug mode this is too verbose
 
+# Remove Artifactory version
+remote_versions="$(grep -v -E "Artifactory/[0-9]+\.[0-9]+\.[0-9]+" <<< "${remote_versions}")"
+
+# Grep the versions
+remote_versions="$(grep -o -E "[0-9]+\.[0-9]+\.[0-9]+(-(rc|beta|alpha|oci)-?[0-9]*)?" <<< "${remote_versions}" | uniq)"
+
+# Sort versions
+if [[ "${TFENV_SORT_VERSIONS_REMOTE:-0}" -eq 1 ]]; then
+  remote_versions="$(sort --version-sort <<< "${remote_versions}")"
+fi
+
+# Reverse versions
 if [[ "${TFENV_REVERSE_REMOTE:-0}" -eq 1 ]]; then
-  grep -o -E "[0-9]+\.[0-9]+\.[0-9]+(-(rc|beta|alpha|oci)-?[0-9]*)?" <<<"${remote_versions}" | uniq | awk '{a[i++]=$0} END {for (j=i-1; j>=0;) print a[j--] }';
-else
-  grep -o -E "[0-9]+\.[0-9]+\.[0-9]+(-(rc|beta|alpha|oci)-?[0-9]*)?" <<<"${remote_versions}" | uniq;
+  remote_versions="$(awk '{a[i++]=$0} END {for (j=i-1; j>=0;) print a[j--] }' <<< "${remote_versions}")"
 fi;
+
+cat <<< "${remote_versions}"
```

Comment:
Please could you end the file with a newline?

Reply:
Done.